apps/rush-lib/etc/examples/Sample_CustomCommandLine.json

```diff
@@ -35,7 +37,7 @@
     "--color": {
       "optionType": "enum",
       "description": "Enable or disable colors in the build log, defaults to 'true'",
-      "supportedCommands": [ "build", "rebuild" ],
+      "associatedCommands": [ "build", "rebuild" ],
```

Comment:
>rebuild [](start = 40, length = 7)

We should have an example of how to associate a custom option with a custom command.

Add check-copyright here.

Reply:
I already added it below

Reply:
It is already associated with --ship

---
In reply to: [148913014](https://github.com/Microsoft/web-build-tools/pull/431#discussion_r148913014) [](ancestors = 148913014)